Cargo.toml

```diff
@@ -99,16 +99,18 @@ ignored = [
 normal = ["workspace-hack"]
 development = ["expect-test", "pretty_assertions"]
 
+[workspace.metadata.dylint]
```

Comment:
The changes in `Cargo.toml` seems not related to this PR. Is it due to merge conflict?